standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -584,6 +584,32 @@ public OpenTxnsResponse openTxns(OpenTxnRequest rqst) throws MetaException {
           LOG.debug("Going to execute update <" + q + ">");
           stmt.execute(q);
         }
+
+        if (rqst.isSetReplPolicy()) {
+          List<String> rowsRepl = new ArrayList<>();
+          List<String> selectRepl = new ArrayList<>();
+
+          for (int i = 0; i < numTxns; i++) {
+            selectRepl.add("select TM_TARGET_TXN_ID from REPL_TXN_MAP where TM_REPL_POLICY = " + quoteString(
```

Comment:
Suggest to use IN clause instead of running several select queries. Shall refer TxnUtils.buildQueryWithINClause.

Reply:
done